backend/packages/Upgrade/src/api/controllers/FeatureFlagController.ts

```diff
@@ -652,4 +653,44 @@ export class FeatureFlagsController {
   ): Promise<Segment> {
     return this.featureFlagService.deleteList(id, request.logger);
   }
+
+  /**
+   * @swagger
+   * /flags/export/json:
```

Comment:
From the ticket description, I don't see `/json` in the end. Have we decided on this change?

Reply:
I kept that to keep `/export/csv` reserved for email export CSV data as we have for experiments exports. But, we discussed this in the meet and we now would have the below endpoints as Dan has mentioned below.